src/test/wasm_compile_known_gcc_test_failures.txt

```diff
@@ -0,0 +1,98 @@
+# Tests which are known to fail from the GCC torture test suite.
+# TODO(sbc): The original home for this file is in the llvm repo once that
```

Comment:
I believe nothing is stopping you from just updating the LLVM repo first (to exactly the contents you have here) before landing this PR.